.github/workflows/readthedocs-pr-links.yaml

```diff
@@ -0,0 +1,16 @@
+name: Read the Docs Pull Request Preview
+on:
+  pull_request_target:
+    types:
+      - opened
```

Comment:
Should `paths` also be specified here? For example, if no files changed in `docs/` or `src/`, would it make sense to add add a preview link? I'm specifically thinking:

```yaml
paths:
  - "docs/"
  - "src/"
  - ".readthedocs.yml"
```

Would it also make sense to run this on `synchronize` as well?

Reply:
To the first question, restricting paths... maybe? But I'd rather not.
The class of PRs which don't impact is very narrow, and it would be easy for this to drift away from our codebase over time or miss things. e.g. We should add `setup.py` and probably `setup.cfg`. If we switch to something like hatch or flit, we'd need `pyproject.toml`, etc etc.
I think applying uniformly is easier, and at _worst_ adds a noisy link to a PR which didn't need it.

I don't think it's necessary to run on `synchronize` because the PR doc build done by RTD gets updated on sync. So the link from the initial PR opening remains accurate.